harness/determined/common/api/logs.py

```diff
@@ -1,41 +1,25 @@
-from typing import Iterable, List, Optional
+import json
 
-from termcolor import colored
+from typing import Iterable, List, Optional
 
 from determined.common import api
 from determined.common.api import bindings
 
 
-def pprint_task_logs(task_id: str, logs: Iterable[bindings.v1TaskLogsResponse]) -> None:
+def pprint_logs(logs: Iterable[bindings.v1TaskLogsResponse]) -> None:
     try:
         for log in logs:
             print(log.message, end="")
     except KeyboardInterrupt:
         pass
-    finally:
-        print(
-            colored(
-                "Task log stream ended. To reopen log stream, run: "
-                "det task logs -f {}".format(task_id),
-                "green",
-            )
-        )
 
 
-def pprint_trial_logs(trial_id: int, logs: Iterable[bindings.v1TrialLogsResponse]) -> None:
+def print_json_logs(logs: Iterable[bindings.v1TrialLogsResponse]) -> None:
     try:
         for log in logs:
-            print(log.message, end="")
+            print(json.dumps(log.to_json(), indent=4))
     except KeyboardInterrupt:
```

Comment:
we also shouldn't bother throwing keyboard interrupts away, here. idk why i did think, i think i didnt know the had handling at a higher level for keyboard interrupts.

Reply:
Resolved in newest commit